src/routes.js

```diff
@@ -12,39 +12,19 @@ const Dashboard = Loadable({
   loading: Loading,
 });
 
-const Bitcoin = Loadable({
-  loader: () => import('./views/Bitcoin/Bitcoin'),
-  loading: Loading,
-});
-
-const Ethereum = Loadable({
-  loader: () => import('./views/Ethereum/Ethereum'),
-  loading: Loading,
-});
-
-const Litecoin = Loadable({
-  loader: () => import('./views/Litecoin/Litecoin'),
-  loading: Loading,
-});
-
-const Monero = Loadable({
-  loader: () => import('./views/Monero/Monero'),
-  loading: Loading,
-});
-
-const XRP = Loadable({
-  loader: () => import('./views/XRP/XRP'),
+const CryptoDetails = Loadable({
+  loader: () => import('./views/CryptoDetails/CryptoDetails'),
```

Comment:
all of the crypto specific loadables shouldn't be needed as they were all quite similar in their structure, just a few differing props so this is even more performant to the bundle size since we have 1 reusable component across all.